src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -2537,6 +2541,83 @@ bool learnFromOperator(BoundBinaryOperator binary)
                 return false;
             }
         }
+
+        protected void VisitBinaryInterpolatedStringAddition(BoundBinaryOperator node)
```

Comment:
Any chance we could factor some of this logic with `VisitInterpolatedStringBase`? They should be very similar (except that we gather more parts in case of binary op).

Reply:
That worked out nicely. Thanks!